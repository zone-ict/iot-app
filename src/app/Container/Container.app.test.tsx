import React from 'react';
import renderer from 'react-test-renderer';
import Container from './Container.app';

describe('Container component', () => {
  it('should render correctly', () => {
    jest.mock('react-redux', () => ({
      Provider: 'Provider',
    }));

    const tree = renderer.create(<Container />);

    expect(tree).toMatchSnapshot();
  });
});
