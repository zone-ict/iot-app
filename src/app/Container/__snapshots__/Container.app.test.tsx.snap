// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container component should render correctly 1`] = `
<PersistGate
  persistor={
    Object {
      "@@observable": [Function],
      "dispatch": [Function],
      "flush": [Function],
      "getState": [Function],
      "pause": [Function],
      "persist": [Function],
      "purge": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <main
    className="css-1tmw1td"
  >
    <div
      className="css-1slw6ji"
    >
      <div
        css={
          Object {
            "> :not([hidden]) ~ :not([hidden])": Object {
              "--tw-space-y-reverse": "0",
              "marginBottom": "calc(2rem * var(--tw-space-y-reverse))",
              "marginTop": "calc(2rem * calc(1 - var(--tw-space-y-reverse)))",
            },
            "display": "flex",
            "flexDirection": "column",
            "padding": "1rem",
          }
        }
      >
        <div>
          <h1
            className="css-lvgsmi"
          >
            Obniz Service
          </h1>
          <div
            className="css-1wjizp4"
          >
            <div
              className="css-njov1c"
            >
              Service Status: Loading...
            </div>
            <button
              className="css-qrgu0p"
              onClick={[Function]}
            >
              Start Service
            </button>
          </div>
        </div>
        <div>
          <h1
            className="css-lvgsmi"
          >
            Temperature
          </h1>
          <div
            className="css-1wjizp4"
          >
            <div
              className="css-njov1c"
            >
              Monitoring Status: Loading...
            </div>
            <button
              className="css-zs6jsq"
              disabled={true}
              onClick={[Function]}
            >
              Start Monitoring
            </button>
          </div>
        </div>
        <div>
          <h1
            className="css-lvgsmi"
          >
            Security Sensor
          </h1>
          <div
            className="css-1wjizp4"
          >
            <button
              className="css-zs6jsq"
              disabled={true}
              onClick={[Function]}
            >
              Start Monitoring
            </button>
            <button
              className="css-zs6jsq"
              disabled={true}
              onClick={[Function]}
            >
              Stop Monitoring
            </button>
          </div>
        </div>
      </div>
    </div>
  </main>
</PersistGate>
`;
